lib/config_skeleton.rb

```diff
@@ -2,11 +2,17 @@
 require 'fileutils'
 require 'frankenstein'
 require 'logger'
-require 'rb-inotify'
 require 'service_skeleton'
 require 'tempfile'
 require 'digest/md5'
 
+begin
+  require 'rb-inotify' unless ENV["DISABLE_INOTIFY"] == "1"
```

Comment:
For consistency with other instances of `ENV["DISABLE_INOTIFY"]`:

```suggestion
  require 'rb-inotify' unless ENV["DISABLE_INOTIFY"]
```

Reply:
Updated